### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by odyslam**
Change-type: minor
Signed-off-by: Odysseas Lamtzidis <odysseas@balena.io>

Added clarification that different environments require different accounts as per the discussion in flodock: https://www.flowdock.com/app/rulemotion/r-beginners/threads/OqKqyaQDBswzc4Oz7e_76Hyq5kw

### CODE REVIEW

**Comment by OdysLam** [comment]
f

**Review by shaunmulligan — APPROVED** [review]
lgtm

**Comment by garethtdavies** [comment]
Maybe we could add a clarification to the Hardware and Software requirements:

* A balenaCloud account for both the [production](https://dashboard.balena-cloud.com) and [staging](https://dashboard.balena-staging.com/) environments

**Comment by hedss** [comment]
@OdysLam Heya. You appear to have two commits for this PR, but because neither of them follow the commit/PR guidelines, balenaCI has rejected them. The guidelines can be found here: https://github.com/balena-io/process/blob/master/process/Commit_and_PR_Guidelines.md

You've written the footer as the body and the body as the footer. :)

File: README.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ environment.
 There are a couple of ways to inform balena CLI that it should use a different
 environment.
 
+*Please note that environments are isolated and thus you can't use the same user account interchangeably. You need to create a different account for each environment, thus before proceeding with the guide ensure that you have an account for balena-staging.*
```
Comment (hedss) [inline README.md:89]: I'd suggest actually having a link for balena-staging here which goes to the https://dashboard.balena-staging.com site, to make it clear.

**Comment by xginn8** [comment]
Rolled this into https://github.com/balena-io/balena-cli-advanced-masterclass/pull/22

**Pull request closed without merging**